src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
 public class WorkWithFile {
+
+    private static final String SEPARATOR = ",";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String RESULT = "result";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
+    private final FileRead fileRead = new core.basesyntax.impl.FileRead();
+    private final FileWrite fileWrite = new core.basesyntax.impl.FileWrite();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> read = fileRead.readFile(new File(fromFileName));
```

Comment:
```suggestion
        List<String> lines = fileRead.readFile(new File(fromFileName));
```
no verbs should be in variable names